packages/target-web3-v1-test/types/v0.6.4/Events.ts

```diff
@@ -45,8 +56,8 @@ export type Event3_uint256 = ContractEventLog<{
   0: string;
 }>;
 export type Event4 = ContractEventLog<{
-  data: [string, string];
-  0: [string, string];
+  data: Events.EventDataStructOutput;
+  0: Events.EventDataStructOutput;
```

Comment:
nit: I prefer keeping these types flat for ease of use during development (would prefer to see that the type is a [string, string] instead of a Events.EventDataStructOutput when hovering over it, for example)